### PR TITLE
Availability scheduler

### PR DESCRIPTION
**Pull request by chrisjobando**
Working Admin & Nonprofit-side Calendars

TODO:
- Fix booking on non-profit side
- Talk about design for booked appointments on admin-side
- Automate email after booking is made

### CODE REVIEW

**Comment by karpawich** [comment]
So much commit

**Comment by chrisjobando** [comment]
> 
> 
> So much commit

This isn't even done yet lol we need to get emails and thats all you bud

**Comment by karpawich** [comment]
> 
> 
> > So much commit
> 
> This isn't even done yet lol we need to get emails and thats all you bud

You know I'm on it, alright their "bud" slow your roll

File: frontend/components/UnderReviewBody.jsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 const UnderViewBody = () => (
-  <LandingBodyMessage width={750}>
-    Congratulations! After careful considerations, we decide that... We enjoyed the meeting
-    with you and believe that our missions align with each other. Lorem ipsum dolor sit amet,
-    consectetur adipiscing elit. Vel interdum facilisi risus donec egestas nisi malesuada eget.
-    Vitae adipiscing sem fermentum leo sollicitudin. Mollis leo nisl id ac lacus vestibulum amet,
-    feugiat ut. Sed leo id semper est in iaculis.
-  </LandingBodyMessage>
+  <>
+    <LandingBodyMessage width={750}>
```
Comment (karpawich) [inline frontend/components/UnderReviewBody.jsx]: Lorem ipsum! Bad!

**Pull request merged**